lib/config/presets/local/index.ts

```diff
@@ -26,6 +26,7 @@ const resolvers = {
   github,
   gitlab,
   local: null,
+  space: local, // TODO: not sure what this means
```

Comment:
```suggestion
  space: local,
```

loading presets from local platform